src/sideshift.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import type {StandardTx, SwapFuncParams} from './checkSwapService'
+
+const {checkSwapService} = require('./checkSwapService.js')
+const js = require('jsonfile')
+const fetch = require('node-fetch')
+
+const confFileName = './config.json'
+const config = js.readFileSync(confFileName)
+
+const SIDESHIFT_CACHE = './cache/XAIRAW.json'
+const PAGE_LIMIT = 500
+
+type SideShiftTransaction = {
+  quoteId: string,
+  depositAddress: {
+    address: string
+  },
+  depositAsset: string,
+  depositMin: number,
+  depositMax: number,
+  settleAddress: {
+    address: string
+  },
+  settleAsset: string,
+  settleAmount: number,
+  createdAt: string
+}
+
+async function doSideShift (swapFuncParams: SwapFuncParams) {
+  return checkSwapService(fetchSideShift,
+    SIDESHIFT_CACHE,
+    'XAI',
+    swapFuncParams)
+}
+
+async function fetchSideShift (swapFuncParams: SwapFuncParams) {
+  if (!swapFuncParams.useCache) {
+    console.log('Fetching SideShift.ai...')
+  }
+  let diskCache = {txs: []}
+  try {
+    diskCache = js.readFileSync(SIDESHIFT_CACHE)
+  } catch (e) {
+    console.log(e)
+  }
+
+  const newTransactions: StandardTx[] = []
+  let offset = 0
+
+  while (1 && !swapFuncParams.useCache) {
+    try {
+      const url = `https://sideshift.ai/api/v1/affiliate/completedOrders?limit=${PAGE_LIMIT}&offset=${offset}&affiliateId=${config.sideShiftAffiliateId}`
+      const options = {
+        method: 'GET',
+        headers: {
+          'affiliateSecret': `${config.sideShiftAffiliateSecret}` // TODO: figure out how to authenticate with affiliateSecret
```

Comment:
No need to use \`template strings\` if template contains nothing but the string variable.

`'affiliateSecret': config.sideShiftAffiliateSecret` is fine.

Even `affiliateSecret: config.sideShiftAffiliateSecret` is valid.